pom.xml

```diff
@@ -125,27 +125,25 @@
         </configuration>
       </plugin>
       <plugin>
-	<groupId>org.codehaus.mojo</groupId>
-	<artifactId>animal-sniffer-maven-plugin</artifactId>
-	<version>1.7</version>
```

Comment:
> The plugin version is inherited from the parent

Oops, I missed that fact.  How about we replace the `<version>` line with something like the following?
`<!-- version is inherited -->`